src/gui/dialogs/outro.cpp

```diff
@@ -28,6 +28,14 @@
 
 #include <cmath>
 
+namespace
+{
+
+// How long text fading should take - currently a hardcoded value.
```

Comment:
Would be nice if this was a Doxygen comment, but that's a very trivial item.

Reply:
It's not an API.